vatic-install.sh

```diff
@@ -1,20 +1,82 @@
-sudo apt-get install -y git python-setuptools python-dev libavcodec-dev libavformat-dev libswscale-dev libjpeg62 libjpeg62-dev libfreetype6 libfreetype6-dev apache2 libapache2-mod-wsgi mysql-server-5.1 mysql-client-5.1 libmysqlclient-dev gfortran 
+export MYSQL_PASSWORD=${MYSQL_PASSWORD:-hail_ukraine}
+export INSTALL_WITH_EXAMPLE_DATA=${INSTALL_WITH_EXAMPLE_DATA:-false}
+export SERVER_NAME=${SERVER_NAME:-localhost}
 
-sudo easy_install -U SQLAlchemy wsgilog pil cython mysql-python munkres parsedatetime argparse
+set -e
+
+if [[ "$INSTALL_WITH_EXAMPLE_DATA" -eq "true" ]]; then
+    echo "(!) Warning: doing aggressive install (assuming empty box just for us and being rude in actions)"
+fi;
+
+sudo apt-get update
+
+# set some mysql password so we can proceed without interactive prompt for it
+sudo debconf-set-selections <<< "mysql-server mysql-server/root_password password $MYSQL_PASSWORD"
+sudo debconf-set-selections <<< "mysql-server mysql-server/root_password_again password $MYSQL_PASSWORD"
+
+sudo apt-get -y install mysql-server
+sudo apt-get install -y git python-setuptools python-dev libavcodec-dev libavformat-dev libswscale-dev libjpeg62 libjpeg62-dev libfreetype6 libfreetype6-dev apache2 libapache2-mod-wsgi mysql-server mysql-client libmysqlclient-dev gfortran
+sudo apt-get install -y libav-tools
+
+sudo easy_install -U SQLAlchemy pillow wsgilog mysql-python munkres parsedatetime argparse
 sudo easy_install -U numpy
 
 git clone https://github.com/cvondrick/turkic.git
-git clone https://github.com/cvondrick/pyvision.git
-git clone https://github.com/cvondrick/vatic.git
+git clone https://github.com/cluePrints/pyvision.git
```

Comment:
This is only needed unless we'll merge https://github.com/cvondrick/pyvision/pull/1